### DISCUSSION TITLE
second monitor

### DISCUSSION THREAD

**Discussion by risidlo** [xournalpp/xournalpp#2926] [Q&A]
Hello,

I have a problem with xournal ++. When I am using two monitors at the same time, xournal writes lines in wrong place. Circa 3 centimetres above the cursor. Any solutions?

Thanks.

**Comment by rolandlo** [comment]
Is this issue #1801?

**Reply by risidlo** [reply]
Yes, thx a lot.

**Reply by Nestak2** [reply]
I have a very similar problem, but it is due to the fact that the underlying Input System stretches the drawing Tablet over all screens. I have also described my problem in this post [here](https://askubuntu.com/questions/1324194/dual-screen-set-up-shows-stylus-position-off-set). Do you have any advice how to solve it? Note that my problem appears only on Ubuntu 20.04, but not on Windows.

**Reply by Nestak2** [reply]
@risidlo I found a 1-click-solution to my problem, as stated [here](https://askubuntu.com/a/1308720/771488)! Just select on log-in that I want to use Wayland instead of xorg. The issue is completely gone for me now!
